__test__/pages/search/searchResults.test.tsx

```diff
@@ -62,7 +62,7 @@ describe("Search Results page", () => {
           results={{ results, aggregations: aggregationsResults }}
         />
       )
-      const selector = screen.getByLabelText("Sort by")
+      const selector = screen.getAllByLabelText("Sort by")[0]
```

Comment:
Do you know why there are multiple elements with this label text?

Reply:
See my note in the PR description about having to render the Sort Select in different sections at different breakpoints.